hw_19042018/person.rb

```diff
@@ -0,0 +1,12 @@
+class Person
+  attr_accessor :name , :gender
+  def text 
+    puts "person"
+  end
+  def input 
+    puts "enter name: "
+    @name = gets.chomp()
+    puts "enter gender: "
+    @gender = gets.chomp()
```

Comment:
Use single quote for normal string.
Use double quote if insert variable to string.